examples/pets/src/2_main.tsx

```diff
@@ -34,7 +40,7 @@ function JazzAndAuth({ children }: { children: React.ReactNode }) {
         <>
             <Jazz.Provider
                 auth={auth}
-                peer="wss://mesh.jazz.tools/?key=pets-example-jazz@gcmp.io"
+                peer={peer}
```

Comment:
Noticed that the `peer` param isn't anymore handled by jazz.

Is that something we removed to simplify the integration with rn?